integration/update_field_test.go

```diff
@@ -671,6 +671,178 @@ func TestUpdateFieldInc(t *testing.T) {
 	})
 }
 
+func TestUpdateFieldMax(t *testing.T) {
```

Comment:
Can we move those cases to compat test and remove this one?

Reply:
Then what's the point of other Tests in this file (like `TestUpdateFieldInc` for an example?

Reply:
I've updated compat tests, but I think that we still need integration tests because compat tests for both handlers are skipped for now because of document comparison.

Reply:
Nevermind, integration tests removed.